chroma_core/services/lustre_audit/update_scan.py

```diff
@@ -106,28 +107,28 @@ def _updates_available(installed_versions, available_versions):
         updates = False
 
         repos = package_report.keys()
-        for package in self.host.server_profile.serverprofilepackage_set.all():
+        for package_name in chain(self.host.server_profile.base_packages, self.host.server_profile.packages):
```

Comment:
Should this be coerced to a `set`?

Reply:
Makes sense to me, there could be duplicates among the two lists.